src/move/move.spec.ts

```diff
@@ -12,22 +12,22 @@ const mockBoard: Board = [
     undefined,
     undefined,
     undefined,
-    { symbol: "A", color: "green" },
-    { symbol: "B", color: "blue" },
-    { symbol: "A", color: "cyan" },
-    { symbol: "B", color: "magenta" },
+    { symbol: "E", color: "blue" },
+    { symbol: "A", color: "yellow" },
+    { symbol: "A", color: "blue" },
+    { symbol: "C", color: "blue" },
```

Comment:
you could have created a new mock board for your new tests, which would have spared you the need to change the existing tests. You are not limited to one mock board per test file!

Reply:
True, but tied to https://github.com/marmelab/kamon/pull/13#discussion_r1173530850, I will probably remove manually edited boards from every test file at some point.

Reply:
I agree, we need to do that